src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,100 @@
+import { NavLink, useParams } from 'react-router-dom';
+import cn from 'classnames';
+import { Person } from '../../types';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = props => {
+  const { people } = props;
+
+  const { slug } = useParams();
+  const noParent = '-';
+
+  const findParent = (parentName: string | null) => {
+    if (parentName) {
+      return people.find(person => person.name === parentName);
+    } else {
+      return undefined;
+    }
```

Comment:
```suggestion
    return people.find(person => person.name === parentName);
```

It will not work the same?